.travis.yml

```diff
@@ -1,9 +1,8 @@
 language: node_js
 node_js:
-  - 0.6
-  - 0.8
```

Comment:
yes but `md5.js` has some requires at the top, so it can't really be compared to md5.min.js which is bundled